pyproject.toml

```diff
@@ -5,9 +5,9 @@ build-backend = "flit_core.buildapi"
 [tool.flit.metadata]
 module = "ninja_extra"
 dist-name = "django-ninja-extra"
-author = "Ezeudoh Tochukwu"
-author-email = "tochukwu.ezeudoh@gmail.com"
-home-page = "https://github.com/eadwinCode/django-ninja-extra"
+author = "Diego Saraiva"
```

Comment:
@jdiego Do you want to be the author of the project? 😅

Reply:
Excuse me... I think I made a mess... I'm really sorry... I edited the wrong pyproject.yml... sorry about that, really sorry... I aml fixing it now

Reply:
That's fine. No hard feelings